t/mime-header.t

```diff
@@ -136,6 +136,7 @@ while (my ($e, $d) = splice @splice, 0, 2) {
     is Encode::decode('MIME-Header', $e) => $d, info("decode default: $e => $d");
 }
 
+no warnings 'once';
```

Comment:
@pali there can be nothing *wrong* with a `no warnings 'once'` by definition. However I have removed it from this PR to expedite the actual needed fix.